packages/account/src/test-utils/launchNode.ts

```diff
@@ -104,6 +131,7 @@ function getFinalStateConfigJSON({ stateConfig, chainConfig }: SnapshotConfigs)
  * @param fuelCorePath - the path to the fuel-core binary. (optional, defaults to 'fuel-core')
  * @param loggingEnabled - whether the node should output logs. (optional, defaults to true)
  * @param basePath - the base path to use for the temporary folder. (optional, defaults to os.tmpdir())
+ * @param includeInitialState - whether to initialise the chain with some default initial state. (optional, defaults to false)
```

Comment:
Could this be a breaking change? Before we included state by default, granted the param isn't breaking but the change in expectation is?